internal/testing/fixture.go

```diff
@@ -150,6 +150,7 @@ func PrepareTest(ctx context.Context, pl *parsing.PackageLoader, env cfg.Context
 		Command:            testBin.Command,
 		Args:               testBin.Args,
 		WorkingDir:         testBin.WorkingDir,
+		Env:                testDef.Driver.Config.Env,
```

Comment:
This doesn't seem consistent; above we use testBin.Args, and here only Config.Env? Where's the merged ones?

Reply:
Fixed with https://github.com/namespacelabs/foundation/commit/5e91f09ab775eae835dbf4eac5dbbc9e1529dcdc
and aligned other places that did not propagate env.